integrationTests/vm/wasm/wasmVM_test.go

```diff
@@ -0,0 +1,205 @@
+package arwen
+
+import (
+	"encoding/hex"
+	"fmt"
+	"io/ioutil"
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go/data/transaction"
+	"github.com/ElrondNetwork/elrond-go/integrationTests/vm"
+	"github.com/ElrondNetwork/elrond-go/process/factory"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestVmDeployWithTransferAndGasShouldDeploySCCode(t *testing.T) {
```

Comment:
test fail on my machine with latest stable version from hera repository (https://github.com/ElrondNetwork/hera/releases)
![image](https://user-images.githubusercontent.com/50499646/66755563-e70bd080-eea0-11e9-93b1-a61d314191b3.png)

Reply:
Likely due to newly added Gas Metering on WABT.  I'll look into it.

Reply:
Yes, but currently we do not use the latest from hera, but an earlier release.